controller/array_action/messages.py

```diff
@@ -66,3 +66,5 @@
 
 SizeOutOfRangeError_message = "requested size is out of limits. requested: {0}," \
                               " max_in_byte: {1}"
+
+SnapshotSourcePoolMismatchError_message = "Snapshot pool : {0} does not match the source volume pool : {1}"
```

Comment:
Remove whitespaces before `:`

Reply:
It's a convention in the error messages (maybe with 2 or 3 exceptions..)

Reply:
in this file I see 29 ` :`, 11 `:` and 12 without colon

Reply:
what do you want me to do and is it relevant for this scope?

Reply:
Personally, I prefer it to be consistent with all messages and according to the English rules.

Reply:
can you open a refactor ticket for it?